drunc_ui/settings/settings.py

```diff
@@ -155,6 +155,8 @@
 KAFKA_ADDRESS = os.getenv("KAFKA_ADDRESS", "kafka:9092")
 
 KAFKA_TOPIC_REGEX = {
+    # ALL matches all topics.
+    "ALL": "^.*$",
```

Comment:
Is this used anywhere?

Reply:
It's used as the default value for `topic` in `messages.html`